tests/integration/kubernetes/run_kubernetes_tests.sh

```diff
@@ -167,6 +167,24 @@ if [ -f "${arch_config_file}" ]; then
 fi
 
 if policy_tests_enabled; then
+	# we are encountering the issue (https://github.com/kata-containers/kata-containers/issues/8407)
```

Comment:
Wouldn't this clean up block be better fitted in [cleanup_nydus_snapshotter()](https://github.com/kata-containers/kata-containers/blob/main/tests/integration/kubernetes/gha-run.sh#L494)? That function will be execute every time the snapshotter is deployed in https://github.com/kata-containers/kata-containers/blob/main/tests/integration/kubernetes/gha-run.sh#L434

The `cleanup_nydus_snapshotter()` will also be executed when the CI job finishes on https://github.com/kata-containers/kata-containers/blob/main/.github/workflows/run-kata-coco-tests.yaml

Reply:
now I move the code to reset_k8s_images and call it in deploy_nydus_snapshotter.